### PR TITLE
Docs: Adding info on the `autoRowSize` option impacting the scrollbar size

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Adds a mention saying that to display the scrollbar in a proper size, the `autoRowSize` option needs to be set to `true` (fixing #4014)
- Adds minor changes to the docs

[skip changelog]

### CODE REVIEW

File: handsontable/src/dataMap/metaManager/metaSchema.js [hunk 1]
```diff
@@ -3320,10 +3315,15 @@ export default () => {
...
+     * By default, the `autoColumnSize` option is set to `undefined`,
+     * but the [`AutoColumnSize`](@/api/autoColumnSize.md) plugin acts as enabled.
+     * To disable the [`AutoColumnSize`](@/api/autoColumnSize.md) plugin completely,
+     * set the `autoColumnSize` option to `false`.
+     *
      * Using the [`colWidths`](#colWidths) option forcibly disables the [`AutoColumnSize`](@/api/autoColumnSize.md) plugin.
      *
      * Read more:
-     * - [`AutoColumnSize`](@/api/autoColumnSize.md)
+     * - [`AutoColumnSize` &#8594;](@/api/autoColumnSize.md)
```
Comment (wszymanski) [inline handsontable/src/dataMap/metaManager/metaSchema.js:3326]: Is this change intentional?

File: handsontable/src/dataMap/metaManager/metaSchema.js [hunk 2]
```diff
@@ -3371,7 +3369,7 @@ export default () => {
      * Using the [`rowHeights`](#rowHeights) option forcibly disables the [`AutoRowSize`](@/api/autoRowSize.md) plugin.
      *
      * Read more:
-     * - [`AutoRowSize`](@/api/autoRowSize.md)
+     * - [`AutoRowSize`&#8594;](@/api/autoRowSize.md)
```
Comment (wszymanski) [inline handsontable/src/dataMap/metaManager/metaSchema.js:3372]: Is this change intentional?

File: handsontable/src/plugins/autoRowSize/autoRowSize.js [hunk 3]
```diff
@@ -18,13 +18,17 @@ const ROW_WIDTHS_MAP_NAME = 'autoRowSize';
...
  * @description
- * This plugin allows to set row heights based on their highest cells.
+ * The `AutoRowSize` plugin allows you to set row heights based on their highest cells.
  *
  * By default, the plugin is declared as `undefined`, which makes it disabled (same as if it was declared as `false`).
  * Enabling this plugin may decrease the overall table performance, as it needs to calculate the heights of all cells to
  * resize the rows accordingly.
  * If you experience problems with the performance, try turning this feature off and declaring the row heights manually.
  *
+ * But, to display Handsontable's [scrollbar](https://handsontable.com/docs/8.0.0/demo-scrolling.html)
```
Comment (wszymanski) [inline handsontable/src/plugins/autoRowSize/autoRowSize.js:28]: It should be much more clear for end-user 👍

File: handsontable/src/dataMap/metaManager/metaSchema.js (hunk 1 above)
Comment (kirszenbaum) [reply handsontable/src/dataMap/metaManager/metaSchema.js:3326]: @wszymanski - yup, a minor thing I spotted and wanted to fix straight away

File: handsontable/src/dataMap/metaManager/metaSchema.js (hunk 2 above)
Comment (kirszenbaum) [reply handsontable/src/dataMap/metaManager/metaSchema.js:3372]: @wszymanski - yup, a minor thing I spotted and wanted to fix straight away

**Review by wszymanski — APPROVED** [review]
